app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] [REQUIRED] Good job so far! The associations between models should be set up correctly according, please implement `foreign key` correctly as below 🙏🏼

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```